hello_wifi/mix.exs

```diff
@@ -70,14 +75,11 @@ defmodule HelloWifi.Mixfile do
      "deps.loadpaths":  ["deps.loadpaths", "nerves.loadpaths"]]
   end
 
+  def kernel_modules("rpi0"), do: ["brcmfmac"]
```

Comment:
Is this necessary for the rpi0? The wireless drivers are built into the kernel for the rpi0 system. I'm not sure if it's correct or not that they're built in, but it's currently the case.